docs/reference/QueryCache.md

```diff
@@ -44,14 +44,15 @@ The `onError` and `onSuccess` callbacks on the QueryCache can be used to handle
 - `defaultOptions` can be overridden by each Query - the global callbacks will **always** be called.
 - `defaultOptions` callbacks will be called once for each Observer, while the global callbacks will only be called once per Query.
 
-## `queryCache.find`
+## `
+`
 
```

Comment:
thank you, but this change looks wrong?

Reply:
Good catch. I accidentally deleted some text while I was writing the commit message. fixed